app/topics/shared/user-tag-input/user-tag-input.component.ts

```diff
@@ -59,7 +59,7 @@ export class UserTagInputComponent implements OnInit, OnChanges {
     for (let user of this.users) {
       if (user.picture === undefined) {
         user.picture = 'loading';
-        this.userService.getPicture(user.email)
+        this.userService.getPicture(user.email, true)
```

Comment:
The boolean parameter is not used in the `getPicture`-function. I think it can be removed.